### PR TITLE
Add functionalities to support ZONE_AWARE

### PR DESCRIPTION
**Pull request by leszko**
Merge after Hazelcast 3.12 is released.

### CODE REVIEW

File: stable/hazelcast-enterprise/templates/role.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{- if .Values.rbac.create -}}
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
```
Comment (mesutcelik) [inline stable/hazelcast-enterprise/templates/role.yaml]: so `Role` is not sufficient to query Resource Type `nodes` then we switched to `ClusterRole`?

**Comment by mesutcelik** [comment]
Are we going to update README to mention about ZONE_AWARE?

File: stable/hazelcast-enterprise/templates/role.yaml (hunk 1 above)
Comment (leszko) [reply stable/hazelcast-enterprise/templates/role.yaml]: You're right, `Role` is related to resouces only within the current namespace. For `nodes`, we need `ClusterRole`.

**Comment by leszko** [comment]
I don't think we need to update Helm Chart README, we already have it in the hazelcast-kubernetes README, so I guess it's enough.

**Comment by leszko** [comment]
@mesutcelik Updated!

**Pull request merged**